src/managers/KeybindManager.hpp

```diff
@@ -63,35 +62,35 @@ class CKeybindManager {
     std::list<SKeybind>                                               m_lKeybinds;
 
   private:
-    std::deque<xkb_keysym_t>  m_dPressedKeysyms;
-    std::deque<int>           m_dPressedKeycodes;
+    std::deque<xkb_keysym_t>     m_dPressedKeysyms;
+    std::deque<int>              m_dPressedKeycodes;
 
-    inline static std::string m_szCurrentSelectedSubmap = "";
+    inline static SubmapOptions* m_szCurrentSelectedSubmap = nullptr;
```

Comment:
it's not a `sz` anymore, it's a `p`

Reply:
I also renamed m_szCurrentSubmap from ConfigManager